addons/crm_iap_lead_enrich/data/message_template.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can rename to mail_data.xml like in other modules, just for naming convention.